### PR TITLE
Add watch-element-resize.js

### PR DESCRIPTION
**Pull request by jonataswalker**
Hi,

Would you please add `watch-element-resize.js` to your servers?

Have a good day.

### CODE REVIEW

**Comment by jimaek** [comment]
No tags https://github.com/jonataswalker/watch-element-resize.js/releases
The bot will fail

**Comment by jonataswalker** [comment]
Indeed. Now there is a release.

File: files/watch-element-resize.js/update.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+{
+  "packageManager": "github",
+  "name": "watch-element-resize.js",
+  "repo": "jonataswalker/watch-element-resize.js",
+  "files": {
+    "basePath": "build/",
+    "include": ["watch-element-resize.min.js"]
```
Comment (jimaek) [inline files/watch-element-resize.js/update.json]: There is no such file here https://github.com/jonataswalker/watch-element-resize.js/tree/1.0.0/build

**Comment by jonataswalker** [comment]
OK, sorry for this mess. Fixed on v1.0.1 - https://github.com/jonataswalker/watch-element-resize.js/tree/1.0.1/build

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge
@jonataswalker Thank you

**Pull request closed without merging**